src/element.ts

```diff
@@ -59,12 +60,17 @@ export interface ElementStyle {
  * Element implements a generic base class for VexFlow, with implementations
  * of general functions and properties that can be inherited by all VexFlow elements.
  *
- * The Element is an abstract class that needs to be subclassed to work. It handles
- * style and text-font properties for the Element and any child elements, along with
- * working with the Registry to create unique ids, but does not have any tools for
- * formatting x or y positions or connections to a Stave.
+ * The Element handles style and font properties for the Element and any child
+ * elements, along with working with the Registry to create unique ids.
+ *
+ * The `text` is a series of unicode characters (including SMuFL codes).
+ * The `textFont` property contains information required to style the text (i.e., font family, size, weight, and style). 
+ * This font family is a comma separated list of fonts. 
+ * The method `measureText` calculates the `textMetrics` and `ẃidth` of the `text`.
+ * The methos `renderText` will render the text using the context provided at the coordinates provided 
```

Comment:
Typo: methos => method

You can use the following:
```
The method `renderText(...)` will render the text using the provided context and coordinates.
```

Reply:
done